www/source/docs/best-practices.html.md.erb

```diff
@@ -303,35 +303,186 @@ Chef Habitat packages may be exported with the Supervisor directly into a [a var
+### Deploy the Chef Habitat bastion on Kubernetes
 
-### Kubernetes exporter
+Complete examples may be downloaded from [this folder](https://www.habitat.sh/docs/examples/kubernetes_hab_bastion/)
 
-When using the Chef Habitat Operator, you can easily convert packages and run them on your Kubernetes cluster using the [Kubernetes exporter](https://kinvolk.io/blog/2017/12/introducing-the-habitat-kubernetes-exporter/):
+To explain how this works, let's break down the hab-bastion.yaml file:
 
-```bash
-$ hab pkg export kubernetes ORIGIN/NAME
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: hab-bastion
+spec:
+  ports:
+  - name: gossip-listener
+    protocol: UDP
+    port: 9638
+    targetPort: 9638
+  - name: http-gateway
+    protocol: TCP
+    port: 9631
+    targetPort: 9631
+  selector:
+    app: hab-bastion
+  clusterIP: None
 ```
 
-### Bare Kubernetes
-
-Users are not required to use the Chef Habitat Operator. Chef Habitat packages exported as containers may be deployed to Kubernetes through the [`kubectl` command](http://kubernetes.io/docs/user-guide/pods/single-container/). Using the [Docker exporter](/docs/developing-packages#pkg-exports) to create a containerized application, the container may be launched like this example:
+This service definition creates a virtual IP (VIP), opening access to the Chef Habitat service that will run on the pod.
+- The habitat gossip port (9638/UDP) listener
+- The habitat http-gateway (9631/TCP) listener
+- the service name becomes available in DNS (as `hab-bastion` or `hab-bastion.namespace-name`, etc) so any pod can find it
 
-```bash
+```yaml
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hab-bastion
+spec:
+    spec:
+      securityContext:
+        fsGroup: 42
+```
+
+This section sets a group ownership for the persistent volume mount point, so that the Habitat supervisor can write to it.  The Habiat user (`hab`) by default has uid `42` and gid `42`
+
+```yaml
+      containers:
+      - name: hab-bastion
+        image: mydockerorigin/hab_bastion:latest
+        args:
+        - '--permanent-peer'
+```
+
+The `image:` line defines you where to get the docker container from. In this case the image is created from a Chef Habitat plan using the `hab pkg export docker` functionality. It runs no service except the Chef Habitat supervisor (hab-sup).
+The argument `--permanent-peer` instructs the supervisor to act as a permanent peer.
+
+```yaml
+        resources:
+          requests:
+            memory: "100Mi"
+            cpu: "100m" # equivalent to 0.1 of a CPU core
+```
+
+Resource requests are important to inform the kubernetes scheduler - without them, some nodes could be overloaded while others are underutilized.
```

Comment:
```suggestion
Resource requests are important because they give instructions to the Kubernetes scheduler--without them, you might overload some nodes while under-using others.
```